src/java/org/apache/cassandra/db/compression/ZstdCompressionDictionary.java

```diff
@@ -38,16 +39,28 @@ public class ZstdCompressionDictionary implements CompressionDictionary, SelfRef
 
     private final DictId dictId;
     private final byte[] rawDictionary;
+    private volatile int checksum;
```

Comment:
why is it `volatile`? the value is never updated.

Reply:
It might be changed to final as well.